### PR TITLE
Option to change already assigned loot

### PR DESCRIPTION
**Pull request by aweinmann**
Option to change already assigned loot to something else or back to unassigned to assign it to a different player (solves wrong log for wrongly assigned loot that gets distributed again)

### CODE REVIEW

**Comment by aweinmann** [comment]
any update on if this will be merged?

**Comment by Road-block** [comment]
I don't understand how this solves the issue of wrongly assigned loot.  

The use case here is this.

ML loots an item and quickly hits ESC (or [x] on the dialog) to mark the item "unassigned" to assign later through trading.  
_(this could be because it's a BoE and the raid doesn't stop to do loot during trash, only does bid callouts on the next boss, or even as a speed running method where ML collects everything and raid does all the bids / assignments through trade after the clear)_

BastionLoot will look through the ML logs to find unassigned loot when they trade an item that has a GP value to another player to prompt for assignment.
Addon then knows the new recipient from monitoring the trade action.

The manual assignment menu assumes the unassigned item that the ML received wasn't claimed / traded to anyone later hence it went to bank or d/e, but the item recipient did not change.

Adding all the ms / os / options back into that limited menu doesn't solve wrongly assigned loot as you can't change the recipient that way.

_I can see a case for adding the "unassigned" menu option back in for the niche case where the ML bids an item for themselves and gets assigned as the recipient with ms or os, then before the 2hour trade window expires decides to a. mark the item back to unassigned, b. trade it to another player while in a raid and as ML still so that c. BastionLoot prompts for a new assignment.
His own points would still need to be fixed manually from the +GP to player menu. That's adding too much complexity for such a niche case._

I don't think I'll accept this change sorry.

TL;DR: BastionLoot Loot Log is meant to service the trade loot use case and not persist for long periods of time.  
I would suggest looking at a specialized loot tracking solution like [Mizus RaidTracker](https://www.curseforge.com/wow/addons/mizusraidtracker) that lets the user edit loot details including recipients (and gets some info for prices from BastionLoot if it's installed along side it)

**Comment by aweinmann** [comment]
We ran sometimes into the issue that due to a miss click or an error of somebody bidding we endet up changing some mainspec to offspec manually (removing the gp by hand) and then change to loot log to reflect this change before exporting for persistence.

**Comment by Road-block** [comment]
> 
> 
> We ran sometimes into the issue that due to a miss click or an error of somebody bidding we endet up changing some mainspec to offspec manually (removing the gp by hand) and then change to loot log to reflect this change before exporting for persistence.

Right but having those menu options added so you can do that graphically, you'd still have no way to fix recipient.  
Seems to me that if you want to do manual changes it's probably simpler to edit the line in the export window before copy-paste (I don't think I have that read-only)
